packages/flutter_driver/test/src/real_tests/extension_test.dart

```diff
@@ -877,6 +880,108 @@ void main() {
     expect(children.single['children'], isEmpty);
   });
 
+  group('extension finders', () {
+    final Widget debugTree = Directionality(
+      textDirection: TextDirection.ltr,
+      child: Center(
+          child: Column(
+        key: const ValueKey<String>('Column'),
+        children: <Widget>[
+          const Text('Foo', key: ValueKey<String>('Text1')),
+          const Text('Bar', key: ValueKey<String>('Text2')),
+          FlatButton(
+            child: const Text('Whatever'),
+            key: const ValueKey<String>('Button'),
+            onPressed: () {},
+          ),
+        ],
+      )),
+    );
+
+    testWidgets('unknown extension finder', (WidgetTester tester) async {
+      final FlutterDriverExtension extension = FlutterDriverExtension(
+          (String arg) async => '', true,
```

Comment:
nit: for new tests, since `extension` is now a langauge keyword, avoid using `extension` as a variable name.

Reply:
Good point, renamed `extension` variable occurences in the file.